code/scripting/api/libs/hud.cpp

```diff
@@ -103,17 +104,33 @@ ADE_FUNC(getHUDConfigShowStatus, l_HUD, "number|string gaugeNameOrIndex", "Gets
 }
 
 ADE_FUNC(setHUDGaugeColor, l_HUD,
-         "number|string gaugeNameOrIndex, [number red, number green, number blue, number alpha]",
-         "Modifies color used to draw the gauge in the pilot config", "boolean", "If the operation was successful")
+         "number|string gaugeNameOrIndex, [number red, number green, number blue, number alpha] | [color Color]",
+	"Modifies color used to draw the gauge in the pilot config",
+	"boolean",
+	"If the operation was successful")
 {
 	int idx = getDefaultGaugeIndex(L);
 	int r = 0;
 	int g = 0;
 	int b = 0;
 	int a = 0;
```

Comment:
I know this isn't your change, but I'm assuming `a` should default to 255 in the event that the script supplies numbers but doesn't supply all four.